qc-runner/src/init.r

```diff
@@ -105,13 +105,16 @@ handle_debug <- function(scdata, config, task_name, sample_id, debug_config) {
     is_debug <- debug_config$step %in% c(task_name, 'all')
     
     if (is_debug) {
+        # variable names used by functions
+        seurat_obj <- scdata
+        num_cells_to_downsample <- 6000
```

Comment:
Not strictly related to this PR, but I found https://github.com/biomage-ltd/pipeline/blob/master/qc-runner/src/numGenesVsNumUmis.r#L36 using 5000. Maybe rather than having a default parameter the downsample number should be set from `init.r`?

Reply:
Yeah me hardcoding it for saving is hacky .... not all the steps have a `num_cells_to_downsample` so we are going to get ugly somewhere